vcd/datasource_vcd_vapp_vm.go

```diff
@@ -383,6 +383,13 @@ func vcdVmDS(vmType typeOfVm) map[string]*schema.Schema {
 			Computed:    true,
 			Description: "VM placement policy ID.",
 		},
+		"security_tags": {
+			Type:        schema.TypeSet,
+			Optional:    true,
```

Comment:
If we only plan to _read_ security tags **in the data source**, I think `Optional` is probably not needed.